map/autoware_lanelet2_map_visualizer/README.md

```diff
@@ -0,0 +1,21 @@
+# autoware_lanelet2_map_visualizer package
+
+This package provides the features of visualizing the lanelet2 maps.
+
+## lanelet2_map_visualization
+
+### Feature
+
+lanelet2_map_visualization visualizes autoware_map_msgs/LaneletMapBin messages into visualization_msgs/MarkerArray.
+
+### How to Run
+
+`ros2 run map_loader lanelet2_map_visualization`
```

Comment:
Is `map_loader` correct?

Reply:
Thank you! No, it is not correct.
I fix the readme in d249479 .